lib/eco/treelexer/lexer.py

```diff
@@ -159,9 +159,17 @@ def match(self, pattern, text, pos=0):
             return self.get_token()
         return None
 
-from incparser.astree import TextNode, BOS, EOS
+from incparser.astree import TextNode, BOS, EOS, MultiTextNode
 from grammar_parser.gparser import MagicTerminal, Terminal, IndentationTerminal
 
+class LPH(object):
```

Comment:
Shouldn't this be `LBPH`?

Reply:
I guess I tried to keep it 3 chars, but this is also fine.